doc/nrf/libraries/networking/aws_iot.rst

```diff
@@ -58,6 +63,8 @@ Generating and provisioning certificates
 There are multiple ways to generate and register these certificates:
 
 * The device key pair and certificate are generated by AWS and downloaded onto the device.
+* You must configure AWS with the ``aws configure`` command to generate the key pair.
+  See `AWS CLI example`_ for the commands on AWS configuration.
```

Comment:
This belongs under `⓵ Setting up AWS and configuring permissions`, and should probably replace `and login as a user with appropriate permissions`.

It's a long time since I looked at this, and when looking at the example we're linking to, I don't remember where to find `aws_access_key_id` and `aws_secret_access_key`. That page doesn't explain this. I can try to find a better link we can refer to instead.

Reply:
Here is the aws doc link for how to authenticate the aws cli: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-authentication.html

It has many different options, and the easiest is far down the list and not recommended to use. The recommended is more work to set up. Have discussed this in the team, and we think it's best to let the user decide what to use.